use_cases.py

```diff
@@ -1,29 +1,38 @@
-from aws.src.database.dynamodb.scripts.asset_table.add_patches_data import main as add_patches_data_dynamodb_main
-from aws.src.database.dynamodb.scripts.asset_table.get_assets import main as get_assets_dynamodb_main
-from aws.src.database.dynamodb.scripts.person_table.get_housing_officers_and_area_managers import \
-    main as get_housing_officers_and_area_managers_main
-from aws.src.database.multi_table.amend_tenure import main as amend_tenure_main
-from aws.src.database.multi_table.find_persons_for_properties import main as find_persons_for_properties_main
+# TODO: Uncomment the method and import you want to use
+# from aws.src.database.dynamodb.scripts.asset_table.patch_asset_with_additional_data import main as patch_asset_with_additional_data_main
+# from aws.src.database.dynamodb.scripts.asset_table.get_assets_by_prop_ref import main as get_assets_by_prop_ref_main
+# from aws.src.database.dynamodb.scripts.asset_table.add_patches_data import main as add_patches_data_dynamodb_main
+# from aws.src.database.dynamodb.scripts.asset_table.get_assets import main as get_assets_dynamodb_main
+# from aws.src.database.dynamodb.scripts.person_table.get_housing_officers_and_area_managers import \
+#     main as get_housing_officers_and_area_managers_main
+# from aws.src.database.multi_table.amend_tenure import main as amend_tenure_main
+# from aws.src.database.multi_table.find_persons_for_properties import main as find_persons_for_properties_main
 
 
-def get_assets_dynamodb():
-    get_assets_dynamodb_main()
+# def get_assets_dynamodb():
+#     get_assets_dynamodb_main()
 
 
-def add_patches_data_dynamodb():
-    add_patches_data_dynamodb_main()
+# def add_patches_data_dynamodb():
+#     add_patches_data_dynamodb_main()
 
 
-def get_housing_officers_and_area_managers():
-    get_housing_officers_and_area_managers_main()
+# def get_housing_officers_and_area_managers():
+#     get_housing_officers_and_area_managers_main()
 
 
-def find_persons_for_properties():
-    find_persons_for_properties_main()
+# def find_persons_for_properties():
+#     find_persons_for_properties_main()
 
 
-def amend_tenure():
-    amend_tenure_main()
+# def amend_tenure():
+#     amend_tenure_main()
+    
+# def get_assets_by_prop_ref():
+#     get_assets_by_prop_ref_main()
+
+# def patch_asset_with_additional_data():
+#     patch_asset_with_additional_data_main()
 
```

Comment:
Should this be committed?
These are just function definitions, not the function calls themselves.

Reply:
Yeah I know, but if you don't comment it out python script acts weird and runs them, found it best to comment it all out and only have the one you need!

Reply:
Thinking about it I think the imports should be left commented for easy access to uncomment and run them, and the functions in the middle should be removed.
In hindsight they're somewhat redundant which is my mistake - you can just call the functions from the imports directly in the `__main__` clause.